### PR TITLE
Tooltip

### LINKED ISSUE #62: Tooltip - Desktop
- [ ] Develop the tooltip for screensizes that use the desktop design 
- [ ] For the specific details about each person, reach out to Aarushi (I think) - if it's not her ask David and Sreya 
- [ ] ***The tooltip shows up when a user hovers over a picture and ONLY ONE should be hoverable at a time
Screen Shot 2022-11-09 at 7 35 05 PM

### PR DESCRIPTION
**Pull request by jhylisa666**
Added ToolTip for main site for every core member

Fixes #62 

Changelist:

- added data for every core member in data.ts
- edited teamcolumn.ts to incorporate the Person state to see who's being hovered
- added ToolTip component

Notes:
- added name because I thought that's very important
- should not render on mobile, but not sure if that's the case yet oop

Screenshots & Screencasts:

https://user-images.githubusercontent.com/92692037/201754375-255e3f79-4958-4c0b-b792-083a44bf00da.mov

### CODE REVIEW

File: shared-ui/components/meet-the-team/tooltip/ToolTip.styles.ts
Comment (jzhzhang66) [inline shared-ui/components/meet-the-team/tooltip/ToolTip.styles.ts]: Can you make an object or something thats like the colors object?
This will make it less prone to string / spelling errors if you check props.team === 'teams.directors'
const teams = {
directors: "Directors",
tech: "Tech",
...
}

File: shared-ui/components/meet-the-team/tooltip/ToolTip.styles.ts
Comment (jzhzhang66) [inline shared-ui/components/meet-the-team/tooltip/ToolTip.styles.ts]: For all of the colors, can you do ${colors.WHATEVERCOLOR} instead of putting the string? Since we already have all the colors stored in the colors const

File: shared-ui/components/meet-the-team/tooltip/ToolTip.styles.ts
Comment (jzhzhang66) [inline shared-ui/components/meet-the-team/tooltip/ToolTip.styles.ts]: Same here as the above comments

File: shared-ui/components/meet-the-team/tooltip/ToolTip.styles.ts
Comment (jzhzhang66) [inline shared-ui/components/meet-the-team/tooltip/ToolTip.styles.ts]: Since you don't have any styles here, remove these and then just use a div

File: shared-ui/components/meet-the-team/tooltip/ToolTip.tsx
Comment (jzhzhang66) [inline shared-ui/components/meet-the-team/tooltip/ToolTip.tsx]: Can you run a formatter on all your files pretty pleaseeee hehe 

also is your pre-commit working? I think that should've formatted this but if not run the shift option f

**Review by jzhzhang66 — CHANGES_REQUESTED** [review]
OMG LISAAA THIS IS SOOOOO GOOD GIRL UR POPPIN OFFFFFFF HOLY CRAP


I left a few small comments and also: 
- at the smaller screensizes that use the desktop design, the tooltip goes off the screen - maybe we can make it more centered or smaller or add logic to specifically make the right columns' tooltips more to the left at this screensize! 
https://user-images.githubusercontent.com/59738880/201776443-ce7e3a7b-50ed-461e-817c-9196168eeac8.mov

- On mobile for some reason all the images disappeared 
Screenshot 2022-11-14 at 4 58 46 PM

**Review by srazh — CHANGES_REQUESTED** [review]
Lisa this looks so good! I second all the changes Judy mentioned. I also am wondering if its possible to make the tooltips responsive based on text inside so for example I noticed Sreya's looks great but Judy wouldn't need as much padding on each side. Sorry should have specified in the design, feel free to disregard if its too trivial:)

**Comment by jhylisa666** [comment]
> Lisa this looks so good! I second all the changes Judy mentioned. I also am wondering if its possible to make the tooltips responsive based on text inside so for example I noticed Sreya's looks great but Judy wouldn't need as much padding on each side. Sorry should have specified in the design, feel free to disregard if its too trivial:)

I'll try my best! But I'm honestly not sure how to do the padding that depends on the length of the name, so that's why I adjused the padding based on the longest name. 
If you have any ideas lmk!!!

File: shared-ui/components/meet-the-team/tooltip/ToolTip.styles.ts
Comment (jzhzhang66) [inline shared-ui/components/meet-the-team/tooltip/ToolTip.styles.ts]: Can you use team.socialOutreach (and same for all the other teams here?)

File: shared-ui/components/meet-the-team/tooltip/ToolTip.styles.ts
Comment (jzhzhang66) [inline shared-ui/components/meet-the-team/tooltip/ToolTip.styles.ts]: This can probably be simplified to padding: 0.4em;

**Review by jzhzhang66 — APPROVED** [review]
LGTM!!! Just left a few comments if you could please fix those before merging!

**Review by srazh — APPROVED** [review]
lgtm!

**Pull request merged**
